install/overlays/libvirt/kustomization.yaml

```diff
@@ -2,10 +2,11 @@ bases:
 - ../../yamls
 nameSuffix: -libvirt
 
-images:
-- name: kustomize.this/cloud-api-adaptor/image:url
-  newName: quay.io/confidential-containers/cloud-api-adaptor-libvirt # change image if needed
-  newTag: latest
+# change image if needed
+# images:
+# - name: quay.io/confidential-containers/cloud-api-adaptor:latest
+#   newName: quay.io/confidential-containers/cloud-api-adaptor
+#   newTag: release-<timestamp>
```

Comment:
nit: libvirt will not be there in release builds. So the newTag can refer to `dev-<timestamp>`

Reply:
also i think it was switched to release-<commit-id>

Reply:
`dev-<commitid>` as per latest changes for the dev builds which includes libvirt :-)